src/Nethermind/Ethereum.Test.Base/TestBlockJson.cs

```diff
@@ -11,7 +11,8 @@ public class TestBlockJson
         public TestBlockHeaderJson[]? UncleHeaders { get; set; }
         public string? Rlp { get; set; }
         public LegacyTransactionJson[]? Transactions { get; set; }
-
+        [JsonProperty("expectException")]
+        private string? ExpectedException2 { set { ExpectedException = value; } }
```

Comment:
Why ExpectedException**2**?

Reply:
sometimes this value is called expectedException, sometimes expectException. It's just a workaround

Reply:
but can we name it as JsonProperties?